src/create-token.js

```diff
@@ -0,0 +1,32 @@
+// @flow
```

Comment:
This currently lives in [fusion-tokens](https://github.com/fusionjs/fusion-tokens/blob/master/src/index.js), but may make more sense living here.  Alternatively, we can let `fusion-core` depend on `fusion-tokens` if we want to minimize breaking changes.

Reply:
Doesn't matter to me, but just don't want to duplicate code :)

If we can do this in a way that doesn't break current non-optional token usage that would be ideal I think.

Reply:
I think it should live here and have fusion-tokens be only common tokens